templates/Bulk_RNA_seq/Templates/biojupies/form.j2

```diff
@@ -0,0 +1,48 @@
+{% extends "biojupies/base.j2" %}
+
+{% block body %}
+{{ super() }}
+<div class="container">
+  <form method="POST" enctype="multipart/form-data">
+      {{ SectionField(
+        name="_primary",
+        section=null,
+        title="Bulk RNA-seq analysis pipeline",
+        subtitle="This pipeline enables you to analyze and visualize your bulk RNA sequencing datasets with an array of  downstream analysis and visualization tools. The pipeline includes: PCA analysis, Clustergrammer interactive heatmap, library size analysis, differential gene expression analysis, enrichment analysis, and L1000 small molecule search.",
```

Comment:
Is this the only thing that changed from the template? If so, you don't need to override the template for this, you can just defined a `SectionField` in your notebook